### PR TITLE
Feature/ms 44/applications redux

### PR DESCRIPTION
**Pull request by ManuelLopezAnido**
Implementar el modelo "Admin" en redux y definir:
Llamados asincrónicos teniendo en cuenta el estado Fetching, Fullfiled y Rejected. Crear la estructura de carptetas del modulo teniendo en cuenta: constants, actions, reducer, thunks. ✓

Generar la implementacion de redux con react para la definición de todos los llamados asincrónicos y reemplazar la lógica utilizada actualmente en los componentes. ✓

### CODE REVIEW

File: src/Components/Shared/ErrorModal/index.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const ErrorModal = ({ showModal, titleText, closeModal, middleText, buttonText }
           <span onClick={closeModal}>&times;</span>
         </div>
         <div className={styles.content}>
-          <span>{middleText}</span>
+          <span>{showModal}</span>
```
Comment (leimeterjoaquin) [inline src/Components/Shared/ErrorModal/index.jsx:16]: Remember that if we change something in shared components, we must make changes on every component that's using it. Let's discuss with the team if this is an improvement or if we should keep it the way it is right now

File: src/redux/applications/actions.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+export const deleteApplicationFulfilled = (id) => ({
+  type: DELETE_APPLICATION_FULFILLED,
+  payload: id
+});
+export const deleteApplicationRejected = (error) => ({
+  type: DELETE_APPLICATION_REJECTED,
+  payload: error
+});
+
+export const errorToDefault = () => ({
```
Comment (leimeterjoaquin) [inline src/redux/applications/actions.js:80]: You set this action but never use it.

File: src/redux/applications/reducer.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      };
+    case DELETE_APPLICATION_REJECTED: {
+      console.log('Error on DELETE from server is: ', action.payload);
+      return {
+        ...state,
+        isLoading: false,
+        error: action.payload
+      };
+    }
+    case ERROR_TO_DEFAULT: {
```
Comment (leimeterjoaquin) [inline src/redux/applications/reducer.js:127]: Same here. This case is never used.

**Review by leimeterjoaquin — CHANGES_REQUESTED** [review]

**Comment by leimeterjoaquin** [comment]
Please name the branch BaSD-MS-44: applications redux

**Review by leimeterjoaquin — APPROVED** [review]

**Review by aquilesmariani — APPROVED** [review]
Please don't forget to remove `console.log` before making any commit.

Still want to review `NoData` component, but let's get this merged ASAP

**Pull request merged**